build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
   }
   dependencies {
     if (JavaVersion.current().isJava7Compatible()) {
-      classpath 'com.netflix.nebula:nebula-publishing-plugin:2.0.0'
+      classpath 'com.netflix.nebula:nebula-publishing-plugin:2.2.+'
```

Comment:
I wonder if we really want to have variable version dependencies?
That could result in unreproducible builds.